src/util/returnUrlHelper.js

```diff
@@ -4,18 +4,32 @@ const generateReturnUrl = ({ queryParams, isMyAccount }) => {
   const {
     publisherConfig: { adyenConfiguration }
   } = store.getState();
+  let queryParamsString;
 
-  if (isMyAccount && adyenConfiguration?.myaccountReturnUrl)
-    return adyenConfiguration.myaccountReturnUrl;
+  if (isMyAccount && adyenConfiguration?.myaccountReturnUrl) {
+    if (queryParams) {
```

Comment:
I was trying to use optional chaining or short circuiting as an alternative here, but unsuccessful - hence if statement. 

If anyone had better idea, that would be appreciated